glances/outputs/glances_stdout_apidoc.py

```diff
@@ -136,7 +136,7 @@ def print_plugin_description(plugin, stat):
         print('Fields descriptions:')
         print('')
         time_since_update = False
-        for field, description in iteritems(stat.fields_description):
+        for field, description in stat.fields_description ():
```

Comment:
As such it is not callable; [the items methods of dict](https://docs.python.org/3/library/stdtypes.html#dict.items) should be used: `stat.fields_description.items()`